nesting.md

```diff
@@ -66,40 +66,39 @@ api.tx.unique.setCollectionPermissions(ACollectionId, {
 });
 ```
 
-After calling the `setCollectionPermissions` method like above, owners of tokens that belong to `collection A` can nest any token from any collection to owned tokens of `collection A`.
+As a consequence of calling `setCollectionPermissions` method as in the example above, owners of `collection A` tokens would be able to nest any token from any collection to any `collection A` token.
 
-A collection owner can restrict the set of collections from which owners are allowed to nest their tokens:
+A collection owner can restrict the set of collections from which owners are allowed to nest their tokens by making a call to setCollectionPermissions:
 ```javascript
 api.tx.unique.setCollectionPermissions(ACollectionId, {
   nesting: {
     OwnerRestricted: [BCollectionId, DCollectionId]
   }
 });
 ```
-Now, owners can nest tokens only from `collection B` or `collection D`.
+As a result, the owners would be able to nest tokens only from `collection B` or `collection D`.
 
-Finally, a collection owner can disable nesting at any time by issuing the `setCollectionPermissions` transaction like this:
+Finally, a collection owner can disable nesting at any time by issuing the `setCollectionPermissions` transaction:
 ```javascript
 api.tx.unique.setCollectionPermissions(ACollectionId, {
   nesting: 'Disabled'
 });
 ```
 
-When a collection owner disables nesting, all existing bundles remain, a token owner can unnest nested tokens from any bundle owned by him. But token owners now are not allowed to create new bundles.
+When a collection owner disables nesting, all existing bundles remain, a token owner can unnest nested tokens from any bundle owned by him. Creating new bundles, however, will not be possible.
 
 The nesting rules can be changed at any time, the existing bundles won’t be affected.
 
 ## Additional info
- * You can’t burn NFT bundles, you have to destruct them first (unnest or burn the nested tokens). When the root NFT has no
-   children (i.e., it became a regular NFT), you can burn it as well.
+ * You can’t burn NFT bundles (i.e. entire branches), you have to deconstruct them first (unnest or burn the nested tokens separately). Once the root NFT has no more children (i.e. when it reverts to a regular NFT) it can then be burned it as well.
```

Comment:
```suggestion
 * You can’t burn NFT bundles (i.e. entire branches), you have to deconstruct them first (unnest or burn the nested tokens separately). Once the root NFT has no more children (i.e. when it reverts to a regular NFT) it can then be burned as well.
```